ci/kind/test-upgrade-theia.sh

```diff
@@ -150,12 +150,41 @@ else
     CLICKHOUSE_FROM_TAG=$THEIA_FROM_TAG
 fi
 
+# theiaTagArray and chOperatorTagArray are expected to be updated together
+# when we upgrade ClickHouse Operator version
+declare -a theiaTagArray=(
+[0]="0.1.0"
+[1]="0.7.0"
+)
+
+declare -a chOperatorTagArray=(
+[0]="0.18.2"
+[1]="0.21.0"
+)
+
+# find the index of tag less than or equal to $THEIA_TAG, then get the
+# corresponding ClickHouse Operator tag
+INDEX=0
+THEIA_TAG=${THEIA_FROM_TAG:1} # strip leading 'v'
```

Comment:
THEIA_TAG -> THEIA_FROM_TAG_NUM or just versionNums? I feel a little bit confusing to use THEIA_TAG for a "from version" which is different from the current theia version

Reply:
done. I didn't get what does xxx_FROM_TAG mean at the begining.

Reply:
It seems you overwrite it? I guess it will fail as we expect to THEIA_FROM_TAG with the leading 'v' at somewhere else.

Reply:
yes I didn't notice that. Thanks for catching it.